src/pages/View/View.page.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import Header from '../../components/Header';
+import HorizontalCard from '../../components/HorizontalCard';
+import useYoutubeApi from '../../components/helpers/useYoutubeApi';
+
+import {
+  Viewpage,
+  InputSearch,
+  MainContent,
+  SideSection,
+  SideVideo,
+  Title,
+  Video,
+} from './View.styled';
+
+function ViewPage() {
+  const { videoId } = useParams();
+  const [value, setValue] = useState('');
+  const { video, loading, listVideoRelated, error, fetchVideo, fetchRelation } =
+    useYoutubeApi();
+
+  const enterPressed = async (event) => {
+    event.preventDefault();
+  };
+
+  useEffect(() => {
+    fetchRelation(videoId);
+    fetchVideo(videoId);
+    console.log('here');
+    if (error !== null) {
+      console.log(error);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
It is almost always wrong to use this rule.  Right here I think you added it to not add `fetchRelation` and `fetchVideo`, the definition of those functions should not be changing, and if they are and triggering the `useEffect` callback to run more times than needed then we need to to something with those functions instead of adding this disable.

Let me know if you have questions on this as this took me a long time to fully wrap my head around this